WSFrontend/org.openl.rules.ruleservice.ws/src/org/openl/rules/ruleservice/publish/jaxrs/JAXRSExceptionMapper.java

```diff
@@ -16,6 +21,25 @@ public Response toResponse(Exception e) {
             dto.getType(),
             dto.getDetail() != null ? dto.getDetail().replaceAll("\t", "    ").split(System.lineSeparator()) : null);
 
+        if (ExceptionType.isServerError(dto.getType())) {
+            Throwable t = e.getCause();
+            while (t != null && t.getCause() != t) {
+                if ((t instanceof OpenLException) && t.getMessage() != null) {

```

Comment:
Why this exception is not logged in the core module? Why logging of these exceptions depends on REST networking?

Reply:
Exception logging moved to the better place